### PR TITLE
Adds ability to toggle keyboard input on and off for 2-in-1s

### PR DESCRIPTION
**Pull request by ghost**
Toggles keyboard input along with the touchpad when using Windows(Option)+prtsc.  This is useful for 2-in-1 applications like tablet mode.

Tested on a Yoga C740 and ASUS Q405UA.

### CODE REVIEW

**Comment by vit9696** [comment]
CC @usr-sse2

File: VoodooPS2Keyboard/VoodooPS2Keyboard.cpp [hunk 1]
```diff
@@ -1478,7 +1478,7 @@ bool ApplePS2Keyboard::dispatchKeyboardEventWithPacket(const UInt8* packet)
     uint64_t now_abs = *(uint64_t*)(&packet[kPacketTimeOffset]);
     uint64_t now_ns;
     absolutetime_to_nanoseconds(now_abs, &now_ns);
-
+
```
Comment (vit9696) [inline VoodooPS2Keyboard/VoodooPS2Keyboard.cpp]: Do not add trailing spaces please.

**Comment by vit9696** [comment]
I am not sure this is a desired feature. I guess it at least needs to be documented. I do not strictly mind, but it looks weird, and the intent is unclear.

File: VoodooPS2Keyboard/VoodooPS2Keyboard.cpp (hunk 1 above)
Comment (ghost) [reply VoodooPS2Keyboard/VoodooPS2Keyboard.cpp]: Corrected, thanks for pointing it out.

**Comment by ghost** [comment]
> I am not sure this is a desired feature. I guess it at least needs to be documented. I do not strictly mind, but it looks weird, and the intent is unclear.

Thanks, I'll update the documentation to add this.  It's definitely desired by myself and I know of a few others who have wanted to disable the keyboard when a 2-in-1 is in tablet mode, this is the normal behavior in Windows but it's sensor triggered.  The intent is simply to be able to use the 2-in-1 with touch input like an ipad without unwanted keyboard input when it's folded as when you're holding a 2-in-1 in tablet mode your fingertips are against the keys.

**Comment by vit9696** [comment]
Hm, but as far as I know it already works fine for @usr-sse2's Lenovo Yoga and is usually implemented in hardware.

**Comment by ghost** [comment]
> Hm, but as far as I know it already works fine for @usr-sse2's Lenovo Yoga and is usually implemented in hardware.

Yeah, same on my Yoga but sadly not on my ASUS.  Thanks for merging!

**Pull request merged**